### PR TITLE
Create propub3r6espa33w.onion.xml

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

**Comment by koops76** [comment]
@Bisaloo See also `ppasset42kropoy6.onion`.

**Comment by Bisaloo** [comment]
@koops76, it's already listed in the top comment.

File: src/chrome/content/rules/propub3r6espa33w.onion.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	
+	Refused:
+		pubapp7v22ykdou3.onion (redirects to clearnet)
+
+-->
+<ruleset name="ProPublica (onion)">
+
+	<target host="propub3r6espa33w.onion" />
+	<target host="www.propub3r6espa33w.onion" />
+
```
Comment (koops76) [inline src/chrome/content/rules/propub3r6espa33w.onion.xml]: See also `pubdrop4dw6rk3aq.onion`.

Comment (Bisaloo) [reply src/chrome/content/rules/propub3r6espa33w.onion.xml]: It's down at the moment. Was it working last time you checked it?

Comment (koops76) [reply src/chrome/content/rules/propub3r6espa33w.onion.xml]: @Bisaloo I did not check any `.onion` domains.

**Comment by jeremyn** [comment]
- [ ] The top comment should include a link to both https://www.propublica.org/nerds/a-more-secure-and-anonymous-propublica-using-tor-hidden-services and https://gist.githubusercontent.com/mtigas/0d49b42fab6f9d2f7e69/raw/2-tor (the first links to the second) to prove that these .onion sites are really associated with propublica.org. For example we could put this text, properly formatted, at the very top:

> See https://www.propublica.org/nerds/a-more-secure-and-anonymous-propublica-using-tor-hidden-services for a discussion about ProPublica's .onion services. In particular, this page links to https://gist.githubusercontent.com/mtigas/0d49b42fab6f9d2f7e69/raw/2-tor which has a list of ProPublica .onion domains.

- [ ] Remove the `redirects` parenthetical comments. They are both unneeded and incorrect (https://ppasset42kropoy6.onion redirects to `www.propub3r6espa33w.onion` , not `propub3r6espa33w.onion` ; and http://pubapp7v22ykdou3.onion redirects to http://www.propub3r6espa33w.onion/tools/ , not `clearnet`, though the ultimate destination is http://www.propublica.org/newsapps/ ).
- [ ] Add a `target` for https://cdn.propub3r6espa33w.onion
- [ ] Add a `target` for https://projects.propub3r6espa33w.onion
- [ ] Add a `target` for https://static.propub3r6espa33w.onion
- [ ] Remove the `securecookie` tag, because these domains don't seem to set cookies.
- [ ] Update [`ruleset-whitelist.csv`](https://github.com/EFForg/https-everywhere/blob/d980b8b4c619459faf6c3f1e60805689bc38ab23/utils/ruleset-whitelist.csv#L2538) for `ProPublica.org.xml`.

**Comment by koops76** [comment]
@jeremyn Better keep `securecookie` anyway in case they will add cookies.

**Comment by koops76** [comment]
@Bisaloo Keep the `securecookie` please. If they do not have any cookies it's guaranteed to be safe, and if they start adding them it's better to secure them.

**Comment by Bisaloo** [comment]
> @Bisaloo Keep the securecookie please. If they do not have any cookies it's guaranteed to be safe, and if they start adding them it's better to secure them.

This is a bad idea. If they add cookies, we have no guarantee that they can be secured without breaking anything. We can't test it so let's not add it.

**Comment by jeremyn** [comment]
@koops76 I argue for very limited `securecookie` use in https://github.com/EFForg/https-everywhere/issues/2907#issuecomment-245113096 , and I describe a potential complicated problem with `securecookie` in https://github.com/EFForg/https-everywhere/pull/7527#issuecomment-262405979. I'm not really interested in re-arguing or reopening that discussion.

**Comment by jeremyn** [comment]
@Bisaloo Merged, thanks.

**Pull request merged**